addOns/postman/src/test/java/org/zaproxy/addon/postman/PostmanParserUnitTest.java

```diff
@@ -153,10 +153,43 @@ static Stream<Arguments> requestBodyTestData() throws URISyntaxException {
                         "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus eu tortor efficitur\n"));
     }
 
+    static Stream<Arguments> variablesTestData() throws URISyntaxException {
+        return Stream.of(
+                arguments(
+                        "{\"item\":{\"name\":\"{{name}}\"}}",
+                        "",
+                        "{\"item\":{\"name\":\"{{name}}\"}}"), // no pair
```

Comment:
Preferable add these comments before the lines (which has more space, otherwise they might end up being split into multiple lines which makes it hard to read).